sunbeam-python/sunbeam/plugins/heat/plugin.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2023 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+import click
+from packaging.version import Version
+
+from sunbeam.plugins.interface.v1.openstack import (
+    OpenStackControlPlanePlugin,
+    TerraformPlanLocation,
+)
+
+LOG = logging.getLogger(__name__)
+
+
+class HeatPlugin(OpenStackControlPlanePlugin):
+    version = Version("0.0.1")
+
+    def __init__(self) -> None:
+        super().__init__(
+            name="heat", tf_plan_location=TerraformPlanLocation.SUNBEAM_TERRAFORM_REPO
+        )
+
+    def set_application_names(self) -> list:
+        """Application names handled by the terraform plan."""
+        apps = ["heat", "heat-mysql-router", "heat-cfn", "heat-cfn-mysql-router"]
+        if self.get_database_topology() == "multi":
+            apps.extend(["heat-mysql", "heat-cfn-mysql"])
```

Comment:
```suggestion
            apps.extend(["heat-mysql"])
```

This is not true since the changes on the PR

Reply:
Good catch, fixed now.